source/android/adaptivecards/src/main/java/io/adaptivecards/objectmodel/AdaptiveCardSchemaKey.java

```diff
@@ -104,6 +106,7 @@
   Language,
   Large,
   Left,
+  Level,
```

Comment:
>Level [](http://example.com/codeflow?start=2&length=5)

This one's not in use anymore i don't think and i should have taken it out. Could you possibly pull this from the shared model while you're in here?

Reply:
Done